tools/ci.psm1

```diff
@@ -593,3 +594,212 @@ function Invoke-AppveyorFinish
         throw $_
```

Comment:
line 405 needs to be updated to

```
Push-Artifact -Path $_ -Name 'CodeCoverage'
```

line 412 needs to be updated to

```
 Push-Artifact $testPackageFullName -Name 'artifacts'
```

line 572

```
Push-Artifact -Path $_ -Name 'artifacts'
```

Reply:
Okay